### PR TITLE
Start building an 'alan' command that wraps the compile and AVM together

### PR DESCRIPTION
**Pull request by dfellis**
Resolves \#203

### CODE REVIEW

**Comment by dfellis** [comment]
Screenshot from 2020-09-09 14-42-40

**Comment by depombo** [comment]
This is amazing! What if we have the `alan` command by default take a `.ln` file to compile and run? `alan test.ln` feels familiar coming from `python test.py` and `node test.js`

**Comment by dfellis** [comment]
> This is amazing! What if we have the `alan` command by default take a `.ln` file to compile and run? `alan test.ln` feels familiar coming from `python test.py` and `node test.js`

You can have a default behavior if no subcommand is specified I think, so that should be fine. I don't want to do that, yet, because compilation is so slow, though?

**Comment by depombo** [comment]
> > This is amazing! What if we have the `alan` command by default take a `.ln` file to compile and run? `alan test.ln` feels familiar coming from `python test.py` and `node test.js`
> 
> You can have a default behavior if no subcommand is specified I think, so that should be fine. I don't want to do that, yet, because compilation is so slow, though?

I think that is okay and we should do it anyway because it is in line with the ethos of this PR/task which is making Alan easier to use, or in other words, reducing the amount of steps to use it

**Review by depombo — APPROVED** [review]

File: avm/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+use std::env;
+
+use clap::{App, SubCommand, crate_name, crate_version};
+
+use crate::benchmark::run::benchmark;
+use crate::compile::compile::compile;
```
Comment (dfellis) [inline avm/src/main.rs:6]: compile compile compile

**Pull request merged**